ethereum/datastructures/src/test/java/tech/pegasys/artemis/datastructures/operations/SignedAggregateAndProofTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.datastructures.operations;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.artemis.datastructures.util.DataStructureUtil;
+import tech.pegasys.artemis.datastructures.util.SimpleOffsetSerializer;
+
+class SignedAggregateAndProofTest {
+  private final DataStructureUtil dataStructureUtil = new DataStructureUtil();
+
+  @Test
+  public void shouldRoundTripViaSsz() {
```

Comment:
These tests are mostly congregated in `DeserilizationTest` file, although with all this keeping tests/constants central vs distributed issue, we're all over the place anyway...

Reply:
I'll follow up and move all those tests to the class test cases where they belong. :)

The reason being, if you change `BeaconBlock` you naturally look for `BeaconBlockTest` and run it before committing to make sure things in the "local area" of your change still work.  If the relevant tests are off hiding in an unrelated test class you wind up having to run the full build to find them which lengthens the feedback cycle.